lib/LedgerSMB/DBObject/Reconciliation.pm

```diff
@@ -162,7 +162,9 @@ sub approve {
     # the user should be embedded into the $self object.
     my $report_id = shift @_;
 
-    my $code = $self->call_dbmethod(funcname=>'reconciliation__report_approve', args=>[$report_id]); # user
+    # Even with https://github.com/ledgersmb/LedgerSMB/issues/838 fixes, array parameters DO NOT WORK
+    #my $code = $self->call_dbmethod(funcname=>'reconciliation__report_approve', args=>[$report_id]); # user
+    my $code = $self->call_dbmethod(funcname=>'reconciliation__report_approve', args=> { in_report_id => $report_id }); # user
```

Comment:
@ylavoie in the original code, it should have said:

```perl
   my $code = $self->call_procedure(....);
```
instead of 
```perl
   my $code = $self->call_dbmethod(....);
```
In that case the `args=>[]` would have worked; I don't think `args=>[]` will ever work for `call_dbmethod()`. (I don't see how to make it work)